File: python/packages/frame_ble/examples/send_reset.py
import asyncio
from frame_ble import FrameBle

async def main():
    frame = FrameBle()

    try:
        await frame.connect()

        # soft reset the Frame
        await frame.send_reset_signal()
        print("Reset sent")

        await frame.disconnect()

    except Exception as e:
        print(f"Not connected to Device: {e}")
        return

if __name__ == "__main__":
    asyncio.run(main())

File: webbluetooth/packages/brilliant-ble/example/send-reset.js
import { BrilliantBle } from 'brilliant-ble';

export async function run() {
  const ble = new BrilliantBle();

  // Connect to Frame
  const deviceId = await ble.connect();

  // Send a reset signal to Frame to reboot it
  await ble.sendResetSignal();

  // Disconnect from Frame
  await ble.disconnect();
};
